public/employers/jsumc.yml

```diff
@@ -0,0 +1,43 @@
+name: Jersey Shore University Medical Center
```

Comment:
Hackensack Meridian Health is the parent and that medical center is under them. I think we should have the parent to keep it consistent.